include/view.h

```diff
@@ -362,6 +362,7 @@ void mappable_disconnect(struct mappable *mappable);
 void view_toggle_keybinds(struct view *view);
 
 void view_set_activated(struct view *view, bool activated);
+void view_set_adaptive_sync_fullscreen(struct view *view);
```

Comment:
Looks like we only call `view_set_adaptive_sync_fullscreen()` from within `view.c`, so no need to declare in header file, just put it higher up than any of the places from which it's called.